Code2Gether-Discord-Bot.Tests/Fakes/FakeProject.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Code2Gether_Discord_Bot.Library.Models;
+using Discord;
+
+namespace Code2Gether_Discord_Bot.Tests.Fakes
+{
+    /// <summary>
+    /// Represents a fake <see cref="Project"/>.This class cannot be inherited.
+    /// </summary>
+    internal sealed class FakeProject : Project
```

Comment:
This class is a POCO and does not need to be faked.

Reply:
Thanks for your review!
I made this class because I thought it'd be a good idea to work only with fake objects in the tests.

Reply:
Thanks for the review!
I made this class because...

> I thought it'd be a good idea to work only with fake objects in the tests.